controllers/ocirepository_controller.go

```diff
@@ -61,6 +61,7 @@ import (
 	"github.com/fluxcd/pkg/version"
 	sourcev1 "github.com/fluxcd/source-controller/api/v1beta2"
 	serror "github.com/fluxcd/source-controller/internal/error"
+	"github.com/fluxcd/source-controller/internal/helm/registry"
```

Comment:
Please remove this dependency, there is no good reason the OCI repo reconciler would depend on a Helm package.